src/dsverifier.cpp

```diff
@@ -1831,6 +1856,74 @@ void check_minimum_phase_delta_domain()
     dsv_msg.show_verification_failed();
 }
 
+/*******************************************************************
+ Function: y_k
+
+ Inputs:
```

Comment:
Can you please document the inputs and outputs?

Reply:
DONE!

Reply:
Yes.